pkg/cloud/services/networking/securitygroups.go

```diff
@@ -37,9 +37,9 @@ const (
 
 // ReconcileSecurityGroups reconcile the security groups.
 func (s *Service) ReconcileSecurityGroups(openStackCluster *infrav1.OpenStackCluster, clusterName string) error {
-	s.scope.Logger().Info("Reconciling security groups", "cluster", clusterName)
+	s.scope.Logger().Info("Reconciling security groups", "name", clusterName)
```

Comment:
Couple of things here: `name` refers to the cluster not the security groups, so would need to be qualified. However...

We actually already added this parameter way back when we created the logger: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/434cfe26c624685cc574cab23e3805add6766dde/controllers/openstackcluster_controller.go#L96-L97

I think we should remove this parameter entirely.